contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.19;
 
 import {ConfirmedOwner} from "../../shared/access/ConfirmedOwner.sol";
 import {TypeAndVersionInterface} from "../../interfaces/TypeAndVersionInterface.sol";
+import {IVRFCoordinatorV2Plus} from "./interfaces/IVRFCoordinatorV2Plus.sol";
```

Comment:
Is this used anywhere

Reply:
Nope, this and the import below were not used - just a leftover from the previous code. Solc linter reported that already so I removed it.